serveur_calcul_python/tests/data_gen/tax_data_store.py

```
@@ -0,0 +1,211 @@
+
+    tax_table,lot_table,association_table,next_letter =generate_geo_dataframes(lat,lons,basic_inputs,letter)
+    
+    return tax_table,lot_table,association_table, next_letter
+
+def generate_geo_dataframes(lat,lons,basic_data,start_lot):
+    points = []
+    polygon_set = []
+    g_no_lot = []
+    g_no_lot_start = start_lot
+    i = ord(g_no_lot_start)
+    for index, lon in enumerate(lons[0:len(basic_data)]):
```

Ruff (0.15.12)

[warning] Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)